src/main/resources/config/liquibase/changelog/20241022120000_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20241022120000" author="bbesrour">
+        <modifyDataType tableName="programming_exercise_build_config" columnName="docker_flags" newDataType="longtext"/>
```

Comment:
Since the `docker_flags` are stored as JSON, could you use a `JSON` column instead? This should be supported by both MySQL and PostgreSQL as far as I know and would guarantee that the content at least represents _some_ valid JSON string.

Reply:
I used the same value we used for the buildPlanConfiguration, If we changed this migration I would have to revert the migration on multiple test servers, which I dont think it's worth the change.